tests/rules/method.js

```diff
@@ -219,6 +220,31 @@ eslintTester.run("method", rule, {
             code: "(e = node.insertAdjacentHTML('beforebegin', '<s>safe</s>'))()",
             parserOptions: { ecmaVersion: 6 },
         },
+        // Typescript support tests
+        {
+            code: "return !!selection && (!this.options.isVisible || (this.options.isVisible as any)(selection as any, ...args))",
```

Comment:
Can you find a shorter test case or split this into multiple shorter ones?

Reply:
done!